src/utils.rs

```diff
@@ -7,7 +7,9 @@ use tokio::sync::broadcast;
 use tokio::task::JoinHandle;
 use tracing::{error, info};
 
+pub mod batch_type;
```

Comment:
Consider renaming this file and moving into `utils`.

Reply:
This is already in utils